app/models/user.rb

```diff
@@ -31,6 +31,30 @@ def sysadmin
     has_role?(:sysadmin)
   end
 
+  def editor=(value)
```

Comment:
Should we leave out the global (not tied to admin set) editor and viewer role getter/setters, since we're attaching those roles to admin sets, and having the global role might be confusing?

Reply:
@martinlovell @mikeapp we made the requested changes. please check again! 🙂